### DISCUSSION TITLE
Cannot connect GitHub repository

### DISCUSSION THREAD

**Discussion by kakkokari-gtyih** [vercel/community#6645] [Account Issue] [answered]
### Question

I had trouble with the integration with GitHub and I tried to disconnect and connect back again to see if it works. But when I try to reconnect, `The repository "<repo name>" is owned by an organization. Please link it to a Vercel team instead.` error has appeared.

![image](https://github.com/vercel/community/assets/67428053/e8de7c62-65db-436c-82db-496ed6d82ae0)

**Comment by amyegan** [comment, marked as answer]
Hey @kakkokari-gtyih. Is that repository actually owned by a GitHub organization, GitLab group, or Bitbucket team? If so, it would need to be linked to a Pro or Enterprise team. Hobby accounts can only link to personal repos.

https://vercel.com/guides/connecting-teams-with-personal-git-accounts

**Reply by kakkokari-gtyih** [reply]
Yes, it is a repository in the Github organization account.
I was able to link and re-link them until just yesterday, but today I was confused because I suddenly could not do it.

Thank you anyway!

**Comment by isaacdarcilla** [comment]
Having the same problem.
